core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -65,6 +65,7 @@
     private DataSize sinkMaxBroadcastBufferSize = DataSize.of(200, Unit.MEGABYTE);
     private DataSize maxPagePartitioningBufferSize = DataSize.of(32, Unit.MEGABYTE);
     private int pagePartitioningBufferPoolSize = 8;
+    private PagePartitioningStrategy pagePartitioningStrategy = PagePartitioningStrategy.MOST_EFFICIENT_PER_PAGE;
```

Comment:
documentation page would be useful

Reply:
I would prefer to avoid documenting this property for now, as there's a chance it may be completely eliminated in a near feature (if we decide to remove row based partitioning and manage to prove that the columar doesn't perform significantly worse under normal or certain corner case circumstances).

Reply:
> I would prefer to avoid documenting this property for now, as there's a chance it may be completely eliminated in a near feature 

Please make it deprecated then

Reply:
Do we usually do that for experimental properties?

Reply:
> Do we usually do that for experimental properties?

We sometimes do it for properties which we expect to remove eventually